dcc-portal-ui/app/scripts/repository/js/repository.js

```diff
@@ -122,6 +122,21 @@
         _ctrl.textFiles.forEach(function(f) {
           Restangular.one('download').get( {'fn':f.name}).then(function(data) {
             f.textContent = data;
+          }).then(function(){
+
+            // Workaround for links in README file on Releases page
+
+            angular.element('.markdown_wrapper').delegate('a', 'click', function(){
+              var _elem = jQuery(this),
+                _href = _elem.attr('href');
+              
+              if(_href.indexOf('@') !== -1){
+                _elem.attr('href', 'mailto:' + _href);
```

Comment:
Clicking on a link multiple times leads to:

![image](https://cloud.githubusercontent.com/assets/2717578/18645502/6c383508-7e7b-11e6-8e20-8926d75ffa0a.png)